firmware/inc/myspi.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MYSPI_H
+#define MYSPI_H
+
+#include <hardware/spi.h>
+#include <pico/stdlib.h>
+#include <stdio.h>
+
+class MySPI {
```

Comment:
Let's make explicit what this class is doing with a comment. Something akin to:
"[Shim](https://stackoverflow.com/a/31225042/3312269) class to make Adafruit's `spi_read` and `spi_write` functions adapted for Arduino compatible with the original Bosch driver."

Let's use Doxygen comments for this:
```
/**
 *  A test class. A more elaborate class description.
 */
class MySPI {
...
```